### PR TITLE
Prac 05 code review request

### PR DESCRIPTION
**Pull request by zackaryjwright-jcu**
Please check formatting, naming and logic
@phillipcluff
@MartinArmstrongAU

### CODE REVIEW

File: prac_03/word_generator.py [hunk 1]
```diff
@@ -10,16 +10,38 @@
...
+        else:
+            valid_format = True
+        return valid_format
+
+
+
+
+main()
+
+
```
Comment (PhillipCluff) [inline prac_03/word_generator.py:44]: pycharm didn't tell say anything about the extra blank lines after the main() call? (sorry i am meant to be reviewing prac_05 sorry my bad

File: prac_05/word_occurrences.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+from operator import itemgetter
```
Comment (PhillipCluff) [inline prac_05/word_occurrences.py:1]: can't see where you used itemgetter

File: prac_05/state_names.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+State names in a dictionary
+File needs reformatting
+"""
+
+# TODO: Reformat this file so the dictionary code follows PEP 8 convention
+STATE_NAMES = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
+               "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
+# print(STATE_NAMES)
+
+state = input("Enter short state: ").lower()
```
Comment (MartinArmstrongAU) [inline prac_05/state_names.py]: Should be .upper() as the key values in the dictionary are all caps.

File: prac_05/state_names.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+while state != "":
+    if state in STATE_NAMES:
+        print(state, "is", STATE_NAMES[state])
+    else:
+        print("Invalid short state")
+    state = input("Enter short state: ").lower()
```
Comment (MartinArmstrongAU) [inline prac_05/state_names.py]: Again, .upper().

File: prac_05/word_occurrences.py [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+
+def main():
+    counted_word = {}
+    sentence = input("Please enter a sentence to be checked: ")
+    sentence_parts = sentence.split()
```
Comment (MartinArmstrongAU) [inline prac_05/word_occurrences.py:7]: Could just be names "words"

File: prac_05/word_occurrences.py [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+
+    for each_word in sentence_parts:
```
Comment (MartinArmstrongAU) [inline prac_05/word_occurrences.py:9]: Convention states that this should be for "word" in "words" in correlation with comment above

File: prac_05/word_occurrences.py [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+        unique_word_count = counted_word.get(each_word, 0)  # Stolen from solution - was using count not get
+        counted_word[each_word] = unique_word_count + 1
+
+    print_spacing = max(len(each_word) for each_word in sentence_parts) # Again shamelessly stolen, i had loop
+
```
Comment (MartinArmstrongAU) [inline prac_05/word_occurrences.py:14]: sentence_parts is NOT converted to a list and sorted here as it should be (from solutions)

**Pull request merged**